content/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -1,39 +1,51 @@
 ---
 title: Circuit Breaking
-description: This task demonstrates the circuit-breaking capability for resilient applications
+description: This task shows you how to configure circuit breaking for connections, requests, and outlier detection.
 weight: 50
 keywords: [traffic-management,circuit-breaking]
 ---
 
 > This task uses the new [v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). The old API has been deprecated and will be removed in the next Istio release. If you need to use the old version, follow the docs [here](https://archive.istio.io/v0.7/docs/tasks/traffic-management/).
 
-This task demonstrates the circuit-breaking capability for resilient applications. Circuit breaking allows developers to write applications that limit the impact of failures, latency spikes, and other undesirable effects of network peculiarities. This task will show how to configure circuit breaking for connections, requests, and outlier detection.
+This task shows you how to configure circuit breaking for connections, requests,
+and outlier detection.
 
 ## Before you begin
 
 * Setup Istio by following the instructions in the
   [Installation guide](/docs/setup/).
 
-*   Start the [httpbin]({{< github_tree >}}/samples/httpbin) sample
-    which will be used as the backend service for our task.
+*   Add the [httpbin]({{< github_tree >}}/samples/httpbin) sample to the mesh:
 
-    If you have enabled [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), do
+    If you have enabled [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), run this command:
 
     {{< text bash >}}
     $ kubectl apply -f @samples/httpbin/httpbin.yaml@
     {{< /text >}}
 
-    otherwise, you have to manually inject the sidecar before deploying the `httpbin` application:
+    otherwise, manually inject the sidecar before deploying the `httpbin` application:
 
     {{< text bash >}}
     $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@)
     {{< /text >}}
 
-## Circuit breaker
+    The `httpbin` application serves as the backend service for this task.
 
-Let's set up a scenario to demonstrate the circuit-breaking capabilities of Istio. We should have the `httpbin` service running from the previous section.
+## About this task
```

Comment:
I've mentioned this in another PR this morning, but I don't like this pattern of adding an About this task section. The text for this section should be at the top of the page before anything else. Why would you want someone reading "Before you begin" before they know what the task is about?